app/channels/test/solutions_channel.rb

```diff
@@ -0,0 +1,21 @@
+module Test
+  class SolutionsChannel < ApplicationCable::Channel
+    def subscribed
+      stream_for solution
+    end
+
+    def submit
+      Test::SolutionsChannel.broadcast_to(solution, {
```

Comment:
@iHiD I'm unsure of what serializer to use here. Should we return the whole serialized submission? Or just the test runs?